distribution/src/scripts/micro-integrator.sh

```diff
@@ -330,6 +330,7 @@ do
     -DenableReadinessProbe=true \
     -DenableLivenessProbe=true \
     -DenableManagementApi=true \
+    -DenableManagementApiV1_1=true \
```

Comment:
Can't we make 1.1 as default and have only one system property to enable management APIs?